_includes/topnav.html

```diff
@@ -12,7 +12,7 @@
                 <span class="icon-bar"></span>
             </button>
             <a class="fa fa-lg navbar-brand no_icon" href="{{base}}/doc/index.html">
-              <img class="no_icon" style="width:150px;" src="{{base}}/images/branding/logo/blue/loopback.svg">&nbsp;<span class="projectTitle"></span></a>
+              <img class="no_icon" style="width:144px;" src="{{base}}/images/branding/logo/blue/loopback.svg">&nbsp;<span class="projectTitle"></span></a>
```

Comment:
is it something we've changed when changing to the new L&F? 
My concern is that changing the hard coded value may not solve the problem for different screen side and resolution.

Reply:
I did add the hard coded value with the new L&F so that it would match v4.loopback.io, but I was running into an issue where the banner would increase in height (the picture attached) when the screen was too small. However, I changed it back to 150px to match again and changed the margins instead. 
<img width="801" alt="screen shot 2018-11-21 at 12 49 37 pm" src="https://user-images.githubusercontent.com/42985749/48863574-d25db000-ed97-11e8-9082-6e765b16147a.png">